build.gradle

```diff
@@ -88,6 +88,16 @@ task travisRelease {
     }
 }
 
+task installGitPreCommitHook(type: Copy) {
+    from 'pre-commit'
+    into '.git/hooks'
+}
+
+task makeGitHooksExecutable { Task t ->
+    t.dependsOn(installGitPreCommitHook)
+    Runtime.getRuntime().exec("chmod -R +x .git/hooks/");
+}
+
```

Comment:
- [ ] - Can you also add "uninstallPrecommit" task? This way we can uninstall easily in case the feature is problematic.